packages/pages/src/common/src/feature/features.ts

```diff
@@ -96,3 +97,34 @@ export const convertTemplateConfigToFeatureConfig = (
 
   return featureConfig;
 };
+
+/**
+ * Converts a {@link RedirectConfigInternal} into a valid single {@link FeatureConfig}.
+ */
+export const convertRedirectConfigToFeatureConfig = (
+  config: RedirectConfigInternal
+): FeatureConfig => {
+  const streamConfig = config.stream || null;
+
+  const featureConfigBase: FeatureConfigBase = {
+    name: config.name,
+    streamId: streamConfig?.$id ?? config.streamId,
+    templateType: "JS",
+  };
+
+  let featureConfig: FeatureConfig;
+  // If the redirectConfig does not reference a stream, assume it's a static feature.
+  if (config.redirectType === "static") {
+    featureConfig = {
+      ...featureConfigBase,
+      staticPage: {},
```

Comment:
We'll need to check with Spruce if you can have static redirect filetypes. I don't think we want that, but I could be wrong.